gtk4/Gir.toml

```diff
@@ -404,33 +404,48 @@ status = "generate"
 name = "Gtk.AboutDialog"
 status = "generate"
 generate_builder = true
+# upstream_needs_fix
+trust_return_value_nullability = true
     [[object.signal]]
     name = "activate-link"
     inhibit = true
+    [[object.function]]
+    name = "get_program_name"
+        [object.function.return]
+        nullable = true
```

Comment:
Please link to an upstream issue/commit for these things, or mention an upstream version where it's fixed. That way we can properly keep track of it and remove the override when possible.

Reply:
Should me much clearer now.